build/scripts/k8s-export-openapi/go.mod

```
@@ -0,0 +1,5 @@
+module github.com/agones/agones/build/scripts/k8s-export-openapi
+
+go 1.22
```

```suggestion
go 1.20
```

Since we aren't yet running 1.22 in the build image.

---

changed it to 1.20, but we have 1.21 in [build/scripts/site-config-update-version/go.mod](https://github.com/googleforgames/agones/blob/05c97b31964eb51772a21ae3d81f8c62330144ca/build/scripts/site-config-update-version/go.mod#L3)

---

Yeah, it's weird it doesn't complain, given:

https://github.com/googleforgames/agones/blob/05c97b31964eb51772a21ae3d81f8c62330144ca/build/build-image/Dockerfile#L26

But everything else is `1.20` so might as well standardise.